README.md

```diff
@@ -0,0 +1,168 @@
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Math Magicians] <a name="about-project"></a>
+
+**[Math Magicians]** is a SPA used to get make calculations and get random math-related quotes.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Html, CSS, Javascript and React.js.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Single Page Application build using React]**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Coming soon]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+To run this project you need:
+Node.js, git , linters (Eslint and Styleint) and React.
+
+--
+Example command:
+
+```
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+  npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+
+```
+ --
+
+### Setup
+
+Clone this repository to your desired folder:
+
+--
+Example commands:
+
+```
+  cd my-folder
+  git clone https://github.com/IsmaelMastronardi/Todo_List.git
+```
+---
+
+### Install
+
+Install this project using git, with the npm I command, Then download the content from "Prerequisites" on this README file, and finally run the project using the command npm start.
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  npm start
+```
+---
+
+### Run tests
+
+To run tests, run the following command:
+
+--
+Example command:
+
+```
+  npx stylelint "**/*.{css,scss}"
+  npx eslint .
+  npm test
+```
+---
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ismael Mastronardi**
+
+- GitHub: [@githubhandle](https://github.com/IsmaelMastronardi)
+- Twitter: [@twitterhandle](https://twitter.com/IsmaMastronardi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ismael-mastronardi-361873271/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add and Remove books]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/IsmaelMastronardi/MicroverseCalculator/issues).
```

Comment:
- [Optional] Please kindly fix the broken link, when I click on the link it directs me to the page not found kindly fix that. Having a broken link in the README file doesn’t look more professional 🙃 😎